qiime/plugin/plugin.py

```diff
@@ -188,19 +188,19 @@ class PluginMethods(PluginActions):
     # TODO is `register` a better name now that functions are the only accepted
     # source (i.e. markdown support is gone)?
     def register_function(self, function, inputs, parameters, outputs, name,
-                          description, help_text=None):
+                          description, help_texts=None):
```

Comment:
When chatting offline last Friday @ebolyen and I opted to go with a separate parameter as inlining seemed strange with requiring nested dicts. It is kind of unclear with no context though.

```python
plugin.methods.register_function(
    ...
    parameters={'timeout': Int},
    ...
    help_texts={'timeout': 'The amount of time to wait before continuing '
                           'execution of the method.',
                'table': 'The table to give the function.'}
)
```

```python
plugin.methods.register_function(
    ...
    parameters={'timeout': {'type': Int,
                            'help_text': 'The amount of time to wait before '
                                         'continuing execution of the '
                                         'method.'}},
    ...
)
```